src/compat/compat.h

```diff
@@ -624,7 +624,92 @@ pg_strtoint64(const char *str)
 						  multixact_freeze_table_age,                                              \
 						  oldestXmin,                                                              \
 						  freezeLimit,                                                             \
-						  multiXactCutoff)
+						  multiXactCutoff,                                                         \
+						  NULL)
+#endif
+
+#if PG15_LT
+#define ExecARUpdateTriggersCompat(estate,                                                         \
+								   resultRelInfo,                                                  \
+								   src_partinfo,                                                   \
+								   dst_partinfo,                                                   \
+								   tupleid,                                                        \
+								   oldtuple,                                                       \
+								   inewslot,                                                       \
+								   recheckIndexes,                                                 \
+								   transtition_capture,                                            \
+								   is_crosspart_update)                                            \
+	ExecARUpdateTriggers(estate,                                                                   \
+						 resultRelInfo,                                                            \
+						 tupleid,                                                                  \
+						 oldtuple,                                                                 \
+						 inewslot,                                                                 \
+						 recheckIndexes,                                                           \
+						 transtition_capture)
+#else
+#define ExecARUpdateTriggersCompat(estate,                                                         \
+								   resultRelInfo,                                                  \
+								   src_partinfo,                                                   \
+								   dst_partinfo,                                                   \
+								   tupleid,                                                        \
+								   oldtuple,                                                       \
+								   inewslot,                                                       \
+								   recheckIndexes,                                                 \
+								   transtition_capture,                                            \
+								   is_crosspart_update)                                            \
```

Comment:
In our code we always set this parameter as `false` so we can simplify this signature removing the `is_crosspart_update` and set `false` for prior pg15 version.

Reply:
With these compat macros we usually adopt the signature of the latest version to create the least friction moving forward and also to make it easy to replace the macro once we no longer need to support the older versions that required the macro.